src/main/ruby/core/truffle/ffi/ffi_struct.rb

```diff
@@ -237,6 +237,11 @@ def initialize(pointer=nil, *spec)
 
       if pointer
         @pointer = pointer
```

Comment:
Does it seem weird to ignore the block in the first case?  Maybe there could be a warning that the block is ignored if given?

Reply:
Done.